lib/logr.rb

```diff
@@ -0,0 +1,45 @@
+require "logger"
+
+require "logr/version"
+require "logr/entry"
+require "logr/json_formatter"
+
+module Logr
+  class Logger
+    def initialize(name, level=::Logger::DEBUG, log_device=STDOUT)
```

Comment:
I can see how `level` behaves. I would make the argument more explicity using keyword arguments. It is also explicit when it's being set on the logger itself.